drivers/volume/volume.go

```diff
@@ -751,3 +751,81 @@ func IsCSIDriverWithoutSnapshotSupport(pv *v1.PersistentVolume) bool {
 	}
 	return true
 }
+
+// GetNodeRegion - get the node region
+func GetNodeRegion() (string, error) {
+	nodes, err := core.Instance().GetNodes()
+	if err != nil {
+		logrus.Errorf("getNodeZones: getting node details failed: %v", err)
+		return "", fmt.Errorf("failed in getting the nodes: %v", err)
+	}
+	for _, node := range nodes.Items {
```

Comment:
Wouldn't this code give the first node always (assuming the node list is populated the same way across every call) or a random node name and its corresponding region as we are returning from the first iteration of the loop itself?